content/en/docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -106,29 +108,64 @@ when the Pod is created, so it is ignored by the scheduler).  Therefore:
  - The DaemonSet controller can make Pods even when the scheduler has not been started, which can help cluster
    bootstrap.
 
-Daemon Pods do respect [taints and tolerations](/docs/concepts/configuration/taint-and-toleration),
-but they are created with `NoExecute` tolerations for the following taints with no `tolerationSeconds`:
+### Scheduled by default scheduler (with ScheduleDaemonSetPods alpha feature)
+
+A DaemonSet ensures that all (or some) nodes run a copy of a pod. Normally, the machine that a pod runs on is selected by the Kubernetes scheduler; however, pods of DaemonSet are created and scheduled by DaemonSet controller who leveraged kube-scheduler’s predicates policy. That introduces the following issues:
+
+ * DaemonSet can not respect Node’s resource changes, e.g. more resources after other Pods exit ([#46935](https://github.com/kubernetes/kubernetes/issues/46935), [#58868](https://github.com/kubernetes/kubernetes/issues/58868))
+ * DaemonSet can not respect Pod Affinity and Pod AntiAffinity ([#29276](https://github.com/kubernetes/kubernetes/issues/29276))
+ * Duplicated logic to respect scheduler features, e.g. critical pods ([#42028](https://github.com/kubernetes/kubernetes/issues/42028)),  tolerant/taint
+ * Hard to debug why DaemonSet’s Pod is not created, e.g. not enough resources; it’s better to have a pending Pods with predicates’ event
+ * Hard to support preemption in different components, e.g. DS and default scheduler
+
+In version 1.11, `ScheduleDaemonSetPods` was introduced as an [alpha feature](/docs/reference/command-line-tools-reference/feature-gates/). With this feature,
+DaemonSet Pods are scheduled by default scheduler, instead of DaemonSet controller.
+
+When this feature is enabled, DaemonSet controller creates all DaemonSet pods upfront no matter whether they can be
+scheduled right away or not; and the `NodeAffinity` term (instead of `.spec.nodeName`) is added to those DaemonSet Pods,
```

Comment:
"no matter whether they can be scheduled right away or not" is ambiguous. 
Need to clarify that DaemonSet controller still respects scheduling policy specified in the spec.